web_form_sticky_header/static/src/js/web_form_sticky_header.js

```diff
@@ -0,0 +1,24 @@
+//-*- coding: utf-8 -*-
+//© 2017 Therp BV <http://therp.nl>
+//License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+openerp.web_form_sticky_header = function(instance)
+{
+    instance.web.FormView.include({
+        start: function() {
+            self = this;
+            return this._super().then(function() {
+                var header = self.$el.find('header')
+                header.css({'background-color': '#ededed;'})
+                header.insertAfter(self.$el.parents().children('.oe_view_manager_header'))
```

Comment:
Please give some comments here on what is happening here, and how this leads to a fixed header.